storage/storage.cpp

```diff
@@ -871,6 +871,7 @@ void Storage::RegisterDownloadedFiles(TCountryId const & countryId, MapOptions o
     /// and we can't stop the process.
     /// TODO: Make the applying process cancellable.
     m_queue.begin()->SetFrozen();
+    NotifyStatusChangedForHierarchy(countryId);
```

Comment:
Could you please explain why you need to notify at this point?

Reply:
because I need to notify observers in order they can change current status in UI.

Reply:
Ok